.github/workflows/track-pr.yml

```diff
@@ -0,0 +1,22 @@
+name: track-pr
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  trackpr:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_PR: ${{ toJSON(github.event.pull_request) }}
+      PR_ID: ${{ github.event.pull_request.node_id }}
```

Comment:
why add both PR_ID and GITHUB_PR? Seems the same?

Reply:
`PR_ID`: ID generated by GitHub for each PR. Ex: `PR_klEOH3CVWs91nT92`
`GITHUB_PR`: Pull request body for the PR. It contains a `title` and a `body`.

Reply:
`GITHUB_PR` also contains the ID, so rather only use that and remove `PR_ID`

Reply:
got it!